src/consts.ts

```diff
@@ -4,13 +4,15 @@ import { PublicKey } from "@solana/web3.js";
 export const MAX_SIGNATURES_PER_CALL = 100;
 export const PROMOTED_POOLS_TESTNET = [
   // new PublicKey("7AdV5E8NznuVjNTT8rNm1MBGn8MNuFN9y9poQVSQ6gjD"), // usdc/tts 1%
-  new PublicKey("4xLSZJwLdkQHGqgyx1E9KHvdMnj7QVKa9Pwcnp1x2mDc"), // USDC/TTS 0.05%
+  //new PublicKey("4xLSZJwLdkQHGqgyx1E9KHvdMnj7QVKa9Pwcnp1x2mDc"), // USDC/TTS 0.05%
+  new PublicKey("GTVKQs8o9D52y9SdwfAXCQSDrrCLosvsP19HgHKugpfw"), // USDC/V2 0.01%
```

Comment:
Send also some `V2` tokens to this address `Fp93x7oYTZftGN8jUPv9qE1dVPEASH33P33ew19JedDV` for testing